tag_test.go

```diff
@@ -110,7 +112,7 @@ func TestTag(t *testing.T) {
 			assert.False(t, tag.isEdit)
 			assert.Equal(t, tag.longNames, []string{"--sep"})
 			assert.Equal(t, tag.sep, ":")
-		case "Required":
+		case "Required", "RequiredNotEnforced":
```

Comment:
what about unittest for RequiredNotEnforced specifically?

Reply:
This is the place

Reply:
I mean different unittests for enforced and not enforced that check the results are different in the relevant area

Reply:
No the idea of this test is that the output in --help for Required and RequiredNotEnforced is the same
There is no logic that is run here.
The logic tests are run above